modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -971,6 +971,83 @@ definition {
 		}
 	}
 
+	@description = "LPS-151676 - Verify that it's possible to edit the relationships of the Custom Object entry (One-to-Many)"
+	@priority = "4"
+	test CanEditOneToManyRelationshipOfCustomObjectEntries {
+		task ("Given some Custom Object entries that contain a Custom Field and Relationship (One-to-many)") {
```

Comment:
This first given block would make more sense as 
Something like "Given a Custom Object with a One To Many self relationship and custom title field"